### PR TITLE
TELCODOCS-1949 - RHACM support for templates changes from partial ful…

### PR DESCRIPTION
**Pull request by slovern**
[TELCODOCS-1949](https://issues.redhat.com//browse/TELCODOCS-1949) - RHACM support for templates changes from partial to full

Version(s): 

- 4.18
- 4.17
- 4.16

Issue: https://issues.redhat.com/browse/TELCODOCS-1949

Link to docs preview:

- [Using hub templates in PolicyGenerator or PolicyGenTemplate CRs](https://82758--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/ztp-using-hub-cluster-templates.html) 
- [Specifying group and site configurations in group PolicyGenerator or PolicyGentemplate CRs](https://82758--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/ztp-using-hub-cluster-templates.html#ztp-specifying-nics-in-pgt-crs-with-hub-cluster-templates_hub-cluster-templates-pgt)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: edge_computing/ztp-using-hub-cluster-templates.adoc [hunk 1]
```diff
@@ -26,14 +26,7 @@ include::snippets/pgt-deprecation-notice.adoc[]
...
 
-include::modules/ztp-using-rhacm-hub-cluster-templates.adoc[leveloffset=+1]
-
-[role="_additional-resources"]
-.Additional resources
-
-* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/{rh-rhacm-version}/html-single/governance/index#hub-templates[{rh-rhacm} support for hub cluster templates in configuration policies]
-
-include::modules/ztp-example-hub-template-functions.adoc[leveloffset=+1]
+* link:https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/{rh-rhacm-version}/html-single/governance/index#template-processing[Template Processing]
```
Comment (Missxiaoguo) [inline edge_computing/ztp-using-hub-cluster-templates.adoc:29]: A small comment. Others look good!
```suggestion
* link:https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/{rh-rhacm-version}/html-single/governance/index#template-processing[RHACM support for template Processing in configuration policies]
```

File: modules/ztp-specifying-nics-in-pgt-crs-with-hub-cluster-templates.adoc [hunk 2]
```diff
@@ -17,7 +17,14 @@ The following example shows you how to use three `ConfigMap` CRs and one `Policy
 
 [NOTE]
 ====
-When you use the `fromConfigmap` function, the `printf` variable is only available for the template resource `data` key fields. You cannot use it with `name` and `namespace` fields.
+There is a link:https://kubernetes.io/docs/concepts/configuration/configmap/#motivation[1 MiB size limit] for `ConfigMap` CRs.
+The effective size for `ConfigMap` CRs is further limited by the `last-applied-configuration` annotation.
```
Comment (mburke5678) [inline modules/ztp-specifying-nics-in-pgt-crs-with-hub-cluster-templates.adoc:21]:
```suggestion
The effective size for the `ConfigMap` CRs is further limited by the `last-applied-configuration` annotation.
```

File: modules/ztp-specifying-nics-in-pgt-crs-with-hub-cluster-templates.adoc [hunk 3]
```diff
@@ -17,7 +17,14 @@ The following example shows you how to use three `ConfigMap` CRs and one `Policy
 
 [NOTE]
 ====
-When you use the `fromConfigmap` function, the `printf` variable is only available for the template resource `data` key fields. You cannot use it with `name` and `namespace` fields.
+There is a link:https://kubernetes.io/docs/concepts/configuration/configmap/#motivation[1 MiB size limit] for `ConfigMap` CRs.
```
Comment (mburke5678) [inline modules/ztp-specifying-nics-in-pgt-crs-with-hub-cluster-templates.adoc:20]: Doc guildelines: [When pointing to a link that exists external to Red Hat’s customer portal documentation...use the <document_heading_name> (<document_source>) format.](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-the-link-attribute)
```suggestion
There is a link:https://kubernetes.io/docs/concepts/configuration/configmap/#motivation[1 MiB size limit] (Kubernetes documentation) for `ConfigMap` CRs.
```

**Comment by mburke5678** [comment]
@slovern A couple of suggestions. Otherwise LGTM.

File: modules/ztp-specifying-nics-in-pgt-crs-with-hub-cluster-templates.adoc (hunk 3 above)
Comment (slovern) [reply modules/ztp-specifying-nics-in-pgt-crs-with-hub-cluster-templates.adoc:20]: Thanks for the review @mburke5678 
I've made both suggested changes.

**Review by skopacz1 — APPROVED** [review]
Merge review LGTM

**Pull request merged**